pom.xml

```diff
@@ -221,7 +221,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <excludedGroups>com.google.cloud.spanner.IntegrationTest</excludedGroups>
+          <exclude>com.google.cloud.spanner.IntegrationTest</exclude>
```

Comment:
https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#excludedGroups

> <excludedGroups>
> (TestNG/JUnit47 provider with JUnit4.8+ only and JUnit5+ provider since 2.22.0) Excluded groups/categories/tags. Any methods/classes/etc with one of the groups/categories/tags specified in this list will specifically not be run.

Was that wrong usage of `excludedGroups` before this PR?

Reply:
Ideally, we want to run integration tests and unit tests (ending in `*ClientTest`) with the `native` profile. For this purpose, we tend to ignore any exclusions that take place in other maven-surefire-plugin configurations within the java-shared-config pom or other child poms: https://github.com/googleapis/java-shared-config/blob/ac44ba11d2da2f649670867c520f90253a8a41ff/pom.xml#L807, so that we can explicitly include these tests. However, with the use of `excludedGroups` in java-spanner-jdbc, this overriding of the exclusions gets ignored, therefore causing no integration tests to run in `native` mode. Changing this to `exclude` fixes this issue.

Reply:
> Was that wrong usage of excludedGroups before this PR?

it seems correct usage to skip the integration test marked with `@Category(ParallelIntegrationTest.class)` in this repository.

<img width="717" alt="Screen Shot 2022-01-18 at 2 47 28 PM" src="https://user-images.githubusercontent.com/28604/150008396-303f92dc-3dc0-4cdf-909d-7bb1e1161303.png">

@mpeddada1 With this change in this PR, does `mvn test` skip the tests marked with `@Category(ParallelIntegrationTest.class)`, such as `ITJdbcSimpleStatementsTest` and `ITJdbcReadOnlyTest`.

Reply:
Good question. Just ran `mvn test` with this change and can confirm that tests annotated with `@Category(ParallelIntegrationTest.class)` are skipped. Only unit tests are executed.

Reply:
Thank you for confirmation.